<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformationWidget</class>
 <widget class="QWidget" name="TransformationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TransformationWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="5,0,30,0,30,20">
     <item>
      <layout class="QHBoxLayout" name="inputLayout" stretch="30,30,10,10,10,10">
       <item>
        <widget class="QPlainTextEdit" name="xCoordEdit">
         <property name="placeholderText">
          <string>xCoord</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="yCoordEdit">
         <property name="placeholderText">
          <string>yCoord</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="zCoordEdit">
         <property name="plainText">
          <string>1</string>
         </property>
         <property name="placeholderText">
          <string>H</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="transformationComboBox">
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startBtn">
         <property name="text">
          <string>Transform</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="visualizeBtn">
         <property name="text">
          <string>Visualize</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="parametersLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="settingLabel">
           <property name="text">
            <string>Scale parameters:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPlainTextEdit" name="firstParameter">
             <property name="placeholderText">
              <string>first parameter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="secondParameter">
             <property name="placeholderText">
              <string>second parameter</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="outputLayout" stretch="33,33,33">
       <item>
        <widget class="QLabel" name="inputLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="transformLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="outputLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="testLabel">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="resultLabel">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
